provisioning/device/src/TransportSettings/ProvisioningClientAmqpSettings.cs

```diff
@@ -35,20 +35,5 @@ public ProvisioningClientAmqpSettings(ProvisioningClientTransportProtocol transp
         /// </para>
         /// </remarks>
         public TimeSpan IdleTimeout { get; set; } = TimeSpan.FromMinutes(2);
-
-        /// <summary>
-        /// A keep-alive for the transport layer in sending ping/pong control frames when using web sockets.
-        /// </summary>
-        /// <seealso href="https://docs.microsoft.com/dotnet/api/system.net.websockets.clientwebsocketoptions.keepaliveinterval"/>
-        public TimeSpan? WebSocketKeepAlive { get; set; }
```

Comment:
Moving this to the more generic TransportSettings class that this class inherits from so that every protocol gets this